addons/towing/functions/fnc_addRopeToVehicle.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Adds rope to vehicle inventory.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * cursorObject call ace_towing_fnc_addRopeToVehicle
+ *
+ * Public: No
+ */
+
```

Comment:
Adding this check should make setting fully changeable at runtime

```suggestion

if (!GVAR(addRopeToVehicleInventory)) exitWith {};
```

Reply:
Only for new vehicles created after setting disabling I suppose? Will it be consistent with existing vehicles which already have rope in inventory?

Reply:
Ofc if rope was already added to vehicles it will stay there. I would not concern about "will it be consistent".

Reply:
what about setting `needRestart` argument then?

Reply:
I don't see a need for it, the mission does not need to be restarted not to have the ropes being added to the vehicles.